.evergreen/.evg.yml

```diff
@@ -121,6 +121,26 @@ functions:
         permissions: public-read
         content_type: ${content_type|application/x-gzip}
 
+  "upload android logs":
```

Comment:
I think this got in here because you rebased after opening the PR.  Generally good practice not to do that.

Reply:
I thought I was in another branch. My mistake.